torch/_dynamo/variables/torch.py

```diff
@@ -450,7 +476,7 @@ def _call_softmax(self, tx, args, kwargs, options):
         dim = args[0] if args else kwargs.get("dim", variables.ConstantVariable(None))
 
         def fake_softmax(input):
-            return variables.TensorVariable.create(
+            return wrap_fx_proxy(
```

Comment:
It's probably not worth it now, but it would be easier to get the `wrap_fx_proxy` refactor in as its own PR that can be quickly reviewed, instead of forcing it to be bundled with a bunch of other substantive changes

Reply:
I think that ship sailed a little, but fwiw there was no code change really, just moves and renames.